src/main.rs

```
@@ -40,11 +44,14 @@ fn main() {
         info!("Starting in development mode");
     }
 
-    let http_request_handler = HttpRequestHandler::new();
+    //TODO: There must be a better way to do error handling
+    let http_request_handler = HttpRequestHandler::new()?;
```

We should at least log an error here, or silently swallow errors in this phase